manifest.json

```diff
@@ -5,6 +5,12 @@
   "description": "__MSG_description__",
   "homepage_url": "https://add0n.com/stylus.html",
   "manifest_version": 2,
+  "applications": {
+    "gecko": {
+      "id": "{a9449967-86c7-4d0e-8285-322cef7c40f2}",
+      "strict_min_version": "42.0"
+    }
+  },
   "icons": {
```

Comment:
Please remove the "applications" section from the manifest... I can't load the extension in chrome with it there.

Reply:
It's there for testing the integration, I'm going to remove it later :p  

That is weird... I can load the extension on chrome even with this section, it just shows me a warning, but it works. 

Chrome version: Version 68.0.3440.106 (Official Build) (64-bit)

Reply:
Ahh, you're right. It still works, but there is an error message

![2018-09-03 21_05_46-extensions](https://user-images.githubusercontent.com/136959/45006182-3d7bbf80-afbd-11e8-81cd-2b0b3e39f6cb.png)